app/views/collections/ugc.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+module Collections
+  class Ugc < ApplicationView
+    include BrowsableView
+    include SearchableView
+
+    def js_vars
+      [
+        {
+          name: 'pageName', value: 'e7a_1418'
+        }
+      ]
+    end
+
+    def page_content_heading
+      if Collection.find_by_key('firstworldwar').present?
+        Collection.find_by_key('firstworldwar').title
```

Comment:
Let's not make the same db query three times...

Looking at `CollectionsController#ugc`, the collection should now be in `@collection`.